src/lint/config/index.test.ts

```diff
@@ -94,7 +96,11 @@ describe("'discoverConfig' function", () => {
       fileLoader.mockReturnValue(
         Promise.resolve("preset: this is not a valid preset")
       );
-      await expect(discoverConfig(fileLoader)).rejects.toThrow();
+      await expect(discoverConfig(fileLoader)).rejects.toThrow(
```

Comment:
It's enough to just test the message, since it is what most consumers (the users of linter) concern about, and if the message is correct we can reasonably believe the details in error is also correct.